packages/manager/apps/pci-private-registry/src/pages/list/IAMColumnLabel.tsx

```diff
@@ -0,0 +1,52 @@
+import {
```

Comment:
Could we consider enhancing the folder structure of `src/pages/list` ? 

I'm thinking about having  `components` & `hooks` folders inside to improve separation of concerns and readability.